heft-plugins/heft-typescript-plugin/src/TypeScriptBuilder.ts

```diff
@@ -976,7 +1031,7 @@ export class TypeScriptBuilder {
     let compilerHost: TTypescript.CompilerHost | undefined;
 
     if (tsconfig.options.incremental) {
-      compilerHost = ts.createIncrementalCompilerHost(tsconfig.options, this._getCachingTypeScriptSystem(ts));
+      compilerHost = ts.createIncrementalCompilerHost(tsconfig.options, ts.sys);
```

Comment:
No longer using the caching?

Reply:
TypeScript caches all of this internally nowadays